images/opentelemetry-collector-contrib/tests/custom-config.yaml

```diff
@@ -89,46 +84,45 @@ extensions:
     size_in_percentage: 5
 
 service:
-  extensions: 
+  extensions:
     - pprof
     - zpages
     - health_check
     - memory_ballast
   pipelines:
     metrics:
-      receivers: 
+      receivers:
         - otlp
         - hostmetrics
-      processors: 
+      processors:
         - resourcedetection
         - k8sattributes
         - batch
-      exporters: 
+      exporters:
         - logging
         - prometheus
         # - datadog
 
     traces:
-      receivers: 
+      receivers:
         - otlp
-      processors: 
+      processors:
         - resourcedetection
         - k8sattributes
         - batch
-      exporters: 
+      exporters:
         - logging
         - zipkin # fails (with warning) without zipkin deployed
-        - jaeger # fails (with warning) without jaeger deployed
```

Comment:
This was removed: https://github.com/open-telemetry/opentelemetry-helm-charts/pull/892